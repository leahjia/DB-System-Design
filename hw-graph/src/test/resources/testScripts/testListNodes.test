# University Map list of nodes testing file.
# Tests that a list of nodes in map can be computed

# Create a graph
CreateGraph map1

# Add nodes
AddNode map1 n1
AddNode map1 n2
AddNode map1 n3

# Finds the nodes of map1
ListNodes map1
