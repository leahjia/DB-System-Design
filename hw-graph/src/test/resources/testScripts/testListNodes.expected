# University Map list of nodes testing file.
# Tests that a list of nodes in map can be computed

# Create a graph
created graph map1

# Add a pair of nodes
added node n1 to map1
added node n2 to map1
added node n3 to map1

# Finds the nodes of map1
the nodes in map1 are: n1 n2 n3
